src/examples/accordions/stepper/StepperLayout.tsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Stepper } from '@zendeskgarden/react-accordions';
+import { Button } from '@zendeskgarden/react-buttons';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const StyledContent = styled.div`
+  display: grid;
+  justify-items: center;
+  margin: ${p => p.theme.space.lg} 0 0 0;
+`;
+
+const StyledBtnRow = styled.div`
+  display: grid;
+  grid-auto-flow: column;
+  grid-gap: ${p => p.theme.space.sm};
+  margin: ${p => p.theme.space.sm} 0;
+`;
+
+const Example = () => {
+  const [step, setStep] = useState(0);
+
+  const horizontalContent = (index: number) => {
```

Comment:
Is there a way this code for rendering content can be more declarative? The horizontal `Stepper` example code from the old website doesn't have `if else` checks.